src/Two/AbstractProvider.php

```diff
@@ -261,6 +268,23 @@ public function userFromToken($token)
         return $user->setToken($token);
     }
 
+    /**
+     * Refresh User's token.
+     *
+     * @param  string  $token
+     * @param  bool  $useScopes
+     * @return array
+     */
+    public function refreshToken($token, $useScopes = false)
+    {
+        $response = $this->getHttpClient()->post($this->getTokenUrl(), [
+            'headers' => ['Content-type' => 'application/x-www-form-urlencoded'],
```

Comment:
Wondering if this is also needed for all other OAuth 2.0 drivers. Did you only test Facebook? Can you check into a few more drivers?

Reply:
Yes, I tested it for Facebook, Google, LinkedIn (LinkedIn App must be partner with LinkedIn), Gitlab and Github. It worked correctly.